components/02-elements/button/button.config.js

```diff
@@ -30,6 +30,13 @@ module.exports = {
         attributes: 'href="#"',
         text: 'I am a link button',
       }
-    }
+    },
+    {
+      name: 'wide',
```

Comment:
rename to `fluid`, b/c there is a way to make it not so `wide`, when you place it in a small container 😉